### PR TITLE
Create SQL-creating_database.md

### PR DESCRIPTION
**Pull request by kaavyakalani26**
The first draft of my markdown file. My file introduces schemas in SQL and steps and tips to create a basic database structure.

### CODE REVIEW

**Comment by keviny15** [comment]
I think each subtopic should be added to the table of contents just for more clarity and organization

**Comment by kaavyakalani26** [comment]
Thats a good suggestion! I will do that.

**Comment by keviny15** [comment]
I also think that some examples would be helpful for readers. Maybe in the How to create tables section it would be helpful for me to see what the table would look like after writing the command!

Also for the redundancy section, I think you can go more in depth about common ways to fix it similar to what we see with design patterns and code smells if possible.

**Comment by kaavyakalani26** [comment]
Thank you for the suggestions!
For the example, do you mean an example of the query needed to create a table? Or that along with the table which would be an empty table with columns?  

Also for the redundancy, I did add links to fix it but I think you are right. I will add a bit of information talking about ways to fix it

**Comment by keviny15** [comment]
> Thank you for the suggestions! For the example, do you mean an example of the query needed to create a table? Or that along with the table which would be an empty table with columns?
> 
> Also for the redundancy, I did add links to fix it but I think you are right. I will add a bit of information talking about ways to fix it

Yeah I meant the resulting table, maybe even some example values to fill the empty table would be helpful too!

**Comment by keviny15** [comment]
Maybe rephrase 4. Avoid designing your schema in such a way that there are attributes that can be null.
Into : Construct your schema with the intention of minimizing the presence of nullable attributes.

Forgot colons on lines 29, 33, 40.

**Comment by keviny15** [comment]
Thank you for making the changes, looks great to me and I learned a lot!

**Comment by kaavyakalani26** [comment]
Thank you for your comments! I will merge the branch now
